astrocut/cutouts.py

```diff
@@ -271,8 +274,15 @@ def fits_cut(input_files, coordinates, cutout_size, correct_wcs=False, extension
         if verbose:
             print("\nCutting out {}".format(in_fle))
 
+        u_fsspec = None
+        fsspec_kw = None
+
+        if "s3://" or "gs://" in in_fle:
+            u_fsspec = True
+            fsspec_kw = fsspec_kwargs
+
         warnings.filterwarnings("ignore", category=wcs.FITSFixedWarning)
-        with fits.open(in_fle, mode='denywrite', memmap=True) as hdulist:
+        with fits.open(in_fle, mode='denywrite', memmap=True, use_fsspec=u_fsspec, fsspec_kwargs=fsspec_kw) as hdulist:
```

Comment:
I need to confirm this in the `astropy` documentation: If `fsspec_kwargs` is called, is `use_fsspec` assumed to be true? It might be redundant to call the `use_fsspec` argument in this case.

Reply:
From `astropy` [documentation](https://docs.astropy.org/en/stable/io/fits/usage/cloud.html#subsetting-fits-files-hosted-in-amazon-s3-cloud-storage): 

> When opening paths with prefix s3:// (Amazon S3 Storage) or gs:// (Google Cloud Storage), [open](https://docs.astropy.org/en/stable/io/fits/api/files.html#astropy.io.fits.open) will automatically default to use_fsspec=True for convenience.

So we can skip using `use_fsspec`.